src/screens/OrganizationDashboard/OrganizationDashboard.spec.tsx

```diff
@@ -160,12 +160,8 @@ describe('Testing Organization Dashboard Screen', () => {
     expect(screen.getByText(t.events)).toBeInTheDocument();
     expect(screen.getByText(t.blockedUsers)).toBeInTheDocument();
 
-    // Upcoming events - Using more flexible matcher
-    await waitFor(() => {
-      expect(
-        screen.getByText(/Event 1/i, { exact: false }),
-      ).toBeInTheDocument();
-    });
+    // Upcoming events - Use a more flexible matcher
+    expect(screen.getByText(/Event 1/i, { exact: false })).toBeInTheDocument();
```

Comment:
Why was this edited?

Reply:
I happened to see a failing test in src/screens/OrganizationDashboard/OrganizationDashboard.spec.tsx. I also asked this question on slack general channel wher peter harrison said "Please fix the test if it is causing other tests to fail".
![Screenshot from 2024-12-26 18-50-58](https://github.com/user-attachments/assets/70b034d0-364a-44c4-ac5a-cc7585776b5a)